packages/web/scripts/sign.js

```diff
@@ -0,0 +1,53 @@
+const execSync = require('child_process').execSync
+const path = require('path')
+
+const run = (cmd) => {
+  console.log(cmd)
+  execSync(cmd)
+}
+
+exports.default = async (configuration) => {
+  const { path: inputFilePath } = configuration
+
+  const inputFilePathNoSpaces = inputFilePath.replace(/ /g, '_')
+  const fileName = path.basename(inputFilePathNoSpaces)
+
+  console.log({ inputFilePath, inputFilePathNoSpaces, fileName })
+
+  // Move file path to one without space since codesign tool does not support space
+  if (inputFilePath !== inputFilePathNoSpaces) {
+    run(`mv '${inputFilePath}' ${inputFilePathNoSpaces}`)
+  }
+
+  // Copy in file to input volume
+  run(`docker cp '${inputFilePathNoSpaces}' codesign-in:/codesign/packages`)
+
+  // Perform codesign
+  // `--security-opt seccomp=unconfirmed` is necessary, see:
+  // https://github.com/linuxserver/docker-nzbhydra2/issues/20#issuecomment-1119688892
+  run(
+    `
+    export USERNAME=esigner_demo && \
+    export PASSWORD=esignerDemo#1 && \
+    export TOTP_SECRET=RDXYgV9qju+6/7GnMf1vCbKexXVJmUVr+86Wq/8aIGg= && \
```

Comment:
assuming these are the ones to be replaced later?

Reply:
Any danger of these leaking if we export and don't unset them after use?
There's probably no easy way to access this session, so all good then.

Reply:
they'll be coming from circleci env vars!